packages/connectivity_plus/pubspec.yaml

```diff
@@ -0,0 +1,21 @@
+name: connectivity_plus_tizen
+description: Tizen implementation of the connectivity_plus plugin.
+version: 1.0.0
+homepage: https://github.com/flutter-tizen/plugins
+
+flutter:
+  plugin:
+    platforms:
+      tizen:
+        dartPluginClass: ConnectivityPlugin
+        pluginClass: ConnectivityPlusTizenPlugin
+        fileName: connectivity_plus_tizen_plugin.h
+
+dependencies:
+  connectivity_plus_platform_interface: ^1.0.0
```

Comment:
Are you going to update this to 1.0.2?

Reply:
Yes, I'll update it together with the sdk version issue!